espnet2/enh/espnet_model.py

```diff
@@ -160,10 +160,15 @@ def forward(
             elif isinstance(criterion, FrequencyDomainLoss):
                 # for the time-frequency domain criterions
                 if criterion.compute_on_mask:
-                    # compute on mask
+                    # compute loss on masks
+                    if noise_ref is not None:
+                        noise_spec = self.encoder(noise_ref.sum(1), speech_lengths)[0]
```

Comment:
Better to update unit test to cover this line.

Reply:
I believe it has been covered by various test in [test/espnet2/enh/test_espnet_model.py](https://github.com/espnet/espnet/blob/master/test/espnet2/enh/test_espnet_model.py):
```python
test_forward_with_beamformer_net[loss_wrappers0-True-1-mask_mse-IBM-True]
test_forward_with_beamformer_net[loss_wrappers0-True-1-mask_mse-IBM-False]
test_forward_with_beamformer_net[loss_wrappers0-True-1-mask_mse-IRM-True]
test_forward_with_beamformer_net[loss_wrappers0-True-1-mask_mse-IRM-False]
...
test_forward_with_beamformer_net[loss_wrappers0-True-1-magnitude-IBM-True] 
test_forward_with_beamformer_net[loss_wrappers0-True-1-magnitude-IBM-False]
...
test_forward_with_beamformer_net[loss_wrappers0-True-1-spectrum-IBM-True]
test_forward_with_beamformer_net[loss_wrappers0-True-1-spectrum-IBM-False]
...
test_forward_with_beamformer_net[loss_wrappers0-True-1-spectrum_log-IBM-True]
test_forward_with_beamformer_net[loss_wrappers0-True-1-spectrum_log-IBM-False]
...
test_forward_with_beamformer_net[loss_wrappers0-True-2-mask_mse-IBM-True]
test_forward_with_beamformer_net[loss_wrappers0-True-2-mask_mse-IBM-False]
test_forward_with_beamformer_net[loss_wrappers0-True-2-mask_mse-IRM-True]
test_forward_with_beamformer_net[loss_wrappers0-True-2-mask_mse-IRM-False]
...
test_forward_with_beamformer_net[loss_wrappers0-True-2-magnitude-IBM-True]
test_forward_with_beamformer_net[loss_wrappers0-True-2-magnitude-IBM-False]
...
test_forward_with_beamformer_net[loss_wrappers0-True-2-spectrum-IBM-True]
test_forward_with_beamformer_net[loss_wrappers0-True-2-spectrum-IBM-False]
...
test_forward_with_beamformer_net[loss_wrappers0-False-3-spectrum_log-IBM-True]
test_forward_with_beamformer_net[loss_wrappers0-False-3-spectrum_log-IBM-False]
```